### PR TITLE
Add MapEditor_CreateTextInfoPanel for ReMap

### PR DESCRIPTION
**Pull request by Julefox**

### CODE REVIEW

File: vscripts/_mapeditor.gnut [hunk 1]
```diff
@@ -203,7 +203,7 @@ void function MapEditor_SpawnDoor(vector org, vector ang, int type, bool gold =
 			DispatchSpawn(d)
 			break;
 		case eMapEditorDoorType.Horizontal:
-			entity d = CreateDoorEntity("prop_dynamic", $"mdl/door/door_256x256x8_elevatorstyle02_animated.rmdl", "survival_door_plain", org, ang)
+			entity d = CreateDoorEntity("prop_dynamic", $"mdl/door/door_256x256x8_elevatorstyle02_animated.rmdl", "survival_door_plainAddCa", org, ang)
```
Comment (r-ex) [inline vscripts/_mapeditor.gnut]: is this change intentional?

Comment (zeex64) [reply vscripts/_mapeditor.gnut]: no fixed

Comment (Julefox) [reply vscripts/_mapeditor.gnut]: oups, sorry gamers
